index.js

```diff
@@ -76,7 +76,7 @@ module.exports = (plugins, opts) => {
 				cb(null, file);
 			})
 			.catch(err => {
-				cb(new gutil.PluginError('gulp-imagemin:', err, {fileName: file.path}));
+				setImmediate(cb, new gutil.PluginError('gulp-imagemin', err, {fileName: file.path}));
```

Comment:
Can you add a `TODO` comment about removing the setImmediate when gulp 4 is targeted?